float64[6] JointState
---
bool ReturnStatus
string ReturnMessage
